core/common/src/main/java/io/quarkus/artemis/core/common/ArtemisUtil.java

```diff
@@ -0,0 +1,13 @@
+package io.quarkus.artemis.core.common;
+
+public class ArtemisUtil {
```

Comment:
Any reason we cannot put this in `runtime`? It is inherited in all other modules.

Reply:
nope, moved, and removed module